openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -235,15 +235,28 @@ export function validateIdentifiers(data) {
 export function initClassificationValidation() {
     initJqueryRepeat();
     const dataConfig = JSON.parse(document.querySelector('#classifications').dataset.config);
+
+    // Prevent form submission on Enter for classification fields
+    $('#select-classification, #classification-value').on('keydown', function(e) {
+        if (e.key === 'Enter') {
+            e.preventDefault();
+            $('#classifications').repeat('add');
+            return false;
+        }
+    });
+
     $('#classifications').repeat({
         vars: {prefix: 'edition--'},
         validate: function (data) {
             if (data.name === '' || data.name === '---') {
-                return error('#classification-errors', '#select-classification', dataConfig['Please select a classification.']);
+                return error('#classification-errors', '#select-classification',
+                    dataConfig['Please select a classification.']);
```

Comment:
It seems like you might have an automatic linter enabled ; be sure to disable that for open library, otherwise we get unintentional edits like these which can make our code review difficult and make our git history/git blame less useful.

```suggestion
                return error('#classification-errors', '#select-classification', dataConfig['Please select a classification.']);
```

Reply:
got it! I shall keep this in mind in future PRs.